crates/js-sys/tests/wasm/JsString.rs

```diff
@@ -58,6 +58,19 @@ fn ends_with() {
     assert_eq!(js.ends_with("to be", 19), true);
 }
 
+#[wasm_bindgen_test]
+fn from_char_code() {
+    let s = "½+¾=";
```

Comment:
Great test, glad to see some non-ascii chars coverage in here.

Reply:
Thanks, MDN 😄